### PR TITLE
Update llama32 vision (mllama) use attention bias

### PR DESCRIPTION
**Pull request by yaoyu-33**
# What does this PR do ?

Fix a previous workaround in llama32 vision to use attention bias in attention calculation.

**Collection**: [multimodal]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/vlm/recipes/mllama_11b.py [hunk 1]
```diff
@@ -134,15 +136,16 @@ def finetune_recipe(
...
-                "*.language_model.*.linear_fc1",
-                "*.language_model.*.linear_fc2",
+                "linear_qkv",
+                "linear_q",
+                "linear_kv",
             ],
+            dim=8,
+            alpha=32,
+            dropout=0.05,
+            dropout_position="pre",
```
Comment (cuichenx) [inline nemo/collections/vlm/recipes/mllama_11b.py:148]: I think it would be good to link the source of these values from meta here

**Review by cuichenx — COMMENTED** [review]
should we update 90B recipes to instruct model and the same lora config as 11B?

File: nemo/collections/vlm/recipes/mllama_11b.py (hunk 1 above)
Comment (yaoyu-33) [reply nemo/collections/vlm/recipes/mllama_11b.py:148]: they don't have this in checkpoint. They have a separate repo/script for this info

**Review by cuichenx — APPROVED** [review]
LGTM Thanks!

**Pull request merged**